docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/30-common-configurations.md

```diff
@@ -0,0 +1,423 @@
+```
+
+The `Secret` can then be referenced in the Kargo configuration.
+
+```yaml
+controller:
+  gitClient:
+    signingKeySecret:
+      name: kargo-git-signing-key
+      type: gpg
+```
+
+:::note
+When using a signing key, the `gitClient.name` and `gitClient.email`
+configuration options must match the name and email associated with the GPG
+key.
+:::
+
+## Argo CD Configuration
+
+Kargo supports a number of Argo CD-related configurations that can be set at
+installation time.
+
+### Disabling the Argo CD Integration
+
+By default, Kargo will enable the Argo CD integration, which configures Kargo
+to work with `Application` resources created by Argo CD. This can be disabled
+as follows:
+
+```yaml
+controller:
+  argocd:
+    integrationEnabled: false
+```
+
+When not enabled, the controller will not watch Argo CD `Application` resources
+and disable Argo CD specific features. Explicitly disabling is preferred if this
+integration is not desired, as it will grant fewer permissions to the controller.
+
+### Argo CD Namespace
+
+By default, Kargo expects Argo CD to be installed to the `argocd` namespace,
+which is also the default namespace it will use for `Application` resources
+if a namespace is not specified in the
+[`argocd-update` Promotion step](../../50-user-guide/60-reference-docs/30-promotion-steps/argocd-update.md).
+
+If you want to use a different default namespace, this can be configured as
+follows:
+
+```yaml
+controller:
+  argocd:
+    namespace: argocd
+```
+
+In certain cases, you may not want Kargo to be able to look up Argo CD resources
+in other namespaces than the configured default. This can be enforced by setting
+the following configuration:
+
+```yaml
+controller:
+  argocd:
+    watchArgocdNamespaceOnly: true
+```
+
+When enabled, the controller will only watch for `Application` resources in the
+namespace specified by `controller.argocd.namespace`.
+
+## Argo Rollouts Configuration
+
+### Disabling the Argo Rollouts Integration
+
+By default, Kargo will enable the Argo Rollouts integration, which configures
+Kargo to work with `Rollout` resources created by Argo Rollouts as part of the
+[verification feature](../../50-user-guide/20-how-to-guides/14-working-with-stages.md#verifications).
+
+This can be disabled as follows:
+
+```yaml
+controller:
+  argoRollouts:
+    integrationEnabled: false
+```
+
+When not enabled, the controller will not reconcile Argo Rollouts `AnalysisRun`
+resources and attempts to verify Stages via `Analysis` will fail. Explicitly
+disabling is preferred if this integration is not desired, as it will grant
+fewer permissions to the controller.
+
+## Resource Management
+
+### Tuning Concurrent Reconciliation Limits
+
+By default, Kargo will reconcile up to 4 resources of the same kind concurrently
+(e.g. 4 `Stage` resources at a time). It is possible to tune this limit by
+setting a new global value, or by setting a limit per resource kind:
```

Comment:
Nit: Not that we follow any one of these in particular, but all of the common style guides (used in the US, at least) agree that 4 should be "four" here.

https://www.masterclass.com/articles/when-to-spell-out-numbers-explained

Not a hill I would die on by any means.